apps/web/src/pages/Departments/utils.ts

```diff
@@ -0,0 +1,15 @@
+export type DepartmentType =
```

Comment:
Were you going to put this in `types.ts` or did you mean to leave an empty file?

Reply:
Oops, yeah I was then decided it was a little unnecessary :sweat: 

Removed file in [remove empty file](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13424/commits/78c8dfdc421078e584b883d9f7f4d74fa0e4cf62)